src/Application/Channels/Commands/CreateChannelCommand.cs

```diff
@@ -89,10 +81,37 @@ public async Task<Guid> Handle(CreateChannelCommand request, CancellationToken c
             entity.RangeRule = "*";
         }
 
+        entity.ActiveRevision = await GetActiveRevision(request, entity, cancellationToken);
+
         _context.Channels.Add(entity);
 
         await _context.SaveChangesAsync(cancellationToken);
 
         return entity.Id;
     }
+
+    private async Task<Revision?> GetActiveRevision(CreateChannelCommand request, Channel entity, CancellationToken cancellationToken)
+    {
+        Revision? revision = null;
+
+        if (entity.RevisionSelectionStrategy == ChannelRevisionSelectionStrategy.UseRangeRule &&
+                    request.ActiveRevisionId is null)
```

Comment:
What if UseRangeRule is false but active revision is not null - is there a validator for that throws an error?